src/Canonical/FeedItem.php

```diff
@@ -1,281 +1,192 @@
 <?php
+
 namespace RZ\MixedFeed\Canonical;
 
+use DateTime;
+use stdClass;
+
 class FeedItem
 {
-    /**
-     * @var string
-     */
-    protected $id;
-    /**
-     * @var string
-     */
-    protected $platform;
-    /**
-     * @var string
-     */
-    protected $author;
-    /**
-     * @var string
-     */
-    protected $link;
-    /**
-     * @var string
-     */
-    protected $title;
+    protected string $id = '';
+
+    protected string $platform = '';
+
+    protected string $author = '';
+
+    protected string $link = '';
+
+    protected string $title = '';
+
     /**
      * @var Image[]
      */
-    protected $images = [];
-    /**
-     * @var string
-     */
-    protected $message;
-    /**
-     * @var \DateTime
-     */
-    protected $dateTime;
-    /**
-     * @var array
-     */
-    protected $tags = [];
-    /**
-     * @var int|null
-     */
-    protected $likeCount;
+    protected array $images = [];
+
+    protected string $message = '';
+
+    protected ?DateTime $dateTime = null;
+
+    /** @var string[] */
+    protected array $tags = [];
+
+    protected ?int $likeCount = null;
+
+    protected ?stdClass $raw = null;
```

Comment:
Canonical items are nice but sometimes, you might need the original response which contains some details that the standard FeeItem doesn't provide (In my MixedFeed use, I had to put back twitter links in tweets).